src/callbacks_step/analysis.jl

```diff
@@ -23,6 +23,13 @@ Additional errors can be computed, e.g. by passing
 `extra_analysis_errors = (:l2_error_primitive, :linf_error_primitive)`
 or `extra_analysis_errors = (:conservation_error,)`.
 
+If you want to omit the computation (to safe compute-time) of the [`default_analysis_errors`](@ref), specify
```

Comment:
Do we have a docstring for `default_analysis_errors`? It looks like Documenter.jl fails because of this

Reply:
Oh that was indeed missing.